tests/plugins/widget/manual/widgetwithspaces.html

```diff
@@ -0,0 +1,24 @@
+<div id="editor"></div>
+
+<script>
+	CKEDITOR.replace( 'editor', {
+        extraAllowedContent: 'span',
+	    extraPlugins: 'spanwidget'
```

Comment:
There are spaces mixed with tabs here (that's why GH renders it like):
![image](https://user-images.githubusercontent.com/1061942/29113496-736aa96c-7cf1-11e7-83d4-2b870dbe8e8e.png)

Please keep the spacing consistent.